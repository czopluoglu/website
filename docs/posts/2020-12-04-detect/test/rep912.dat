01011101111001000111
00010001010000001110
00010011110011111111
00011111011110001011
00010011101001000111
00100000000000100011
11000011001010001001
00011111111111010111
11010011110001001010
00011100010111011011
10011011111100111111
11110111111100101111
10110011011010100110
01001101011010111010
10000010101001110110
10000111011100110111
11001011001110101111
11001111011110110101
11111111111011100000
11001010011010101111
01001000111000100110
11101001000000110000
00110001001000000010
11111101101011100110
00011010111011110110
10110000010110000111
00000001101011010111
01011111110000111010
01100111011100110001
10110010111000101001
00111101001111110111
11110010100010000010
00110001111001101110
11000001010110000100
00101110100011100010
00100000001110001110
11111111101010100000
10011001100100000000
01111101110110101111
11110011010011000110
01000011001000000101
00011111111011100111
11111111111111101111
10001000000001100101
01110001000110000010
10001001110110011111
01000000001111101101
10111101101111101010
01011111111111011111
11110001010000111111
11110001000001010011
00001100011111110010
00011000011111111010
01000101100110001011
11100000001011110101
00111101000111111111
01110111011110101111
11110011101101101011
11101111011111101101
01100001000000000111
00011010001000000001
01100001101111111111
11110101110001101100
11110001011111110111
11011111101111110111
11100111111101010011
11110111110101111110
11110110011010101110
11101010111111111011
00101000001010010110
00111001101110101010
10101001000100000011
01101111000010110011
10100000111110100010
00010000110000000010
11001001101000000001
11000011111000001011
11100100010000001111
10100001001100100101
11001001111110101111
10011110111111110110
11111010111010010000
10111100111001011011
11111101001100011010
10111010111111011110
10110000111010111010
10000000010000100010
10101111111100100111
10110001000011100011
11111000111100111101
00011100011000100100
10111001010001101010
01100000001110110111
10011000101110101110
11111101101000110011
01000100000110100111
00010001101000011001
10111011010000100110
00111011101110100010
11001100011111000111
01111101010110111111
00000001100011100110
00011000011100100010
11110111110110100011
11100001000000101111
01010101111100110111
11111111011101110110
01011101111110101100
10101101001001000010
01110111001111111111
11000101111100101110
10110111111010110100
11000001111011001111
10010110111001001011
11010111111111101111
11110111110110111111
11101111111111100111
00001000101100111101
00011011111111011111
00101011111100110111
01111111100101001111
11110010100111011101
00111010011010100001
10110010111011011110
01101001001000110000
01000101111010010110
10000000001100100111
11000101111101100011
00100000010110100001
01000010011101000111
10101101000010101100
10111111111011011011
10111111111110110110
10100101111010111111
11111111111111110110
00111100111000011111
01101011001111101111
00101101011010100010
10010101011110111011
11001010011001000011
11011110011010100010
10000000100010100111
00101101001011111100
10000100001110101111
10100101110111011011
11101001101110101111
01100111011111111110
10100111000000100100
10000001011010000111
00110000001111001001
11011111011010110110
00100111111111100110
00001000100100001010
01000111100110000111
11111110110000000111
00000010011111111111
01000010001000111111
11000100101110111110
10000001001110101111
10111110111010101111
10001110111011000100
11000100101110100010
11110111111011000011
00001111000111100110
11100011001000100011
10001001101000100000
00110001010110110011
00000001110010101011
11000100001010101011
11111111010110000011
11111111111011111111
01011001010010100110
10000100100010100110
00000001001010101010
00000001011010000111
11110110111110011110
01001000011010100110
10000000000101110011
11101111011110111111
01110001110101010101
01001101111111010111
11111111000000100010
11001001101000100001
01001001001011100100
01001001011101111110
10000111011110011110
00011111010101100101
10010001000001111010
00010010010010010010
10010001101001000101
00111100010010110111
00000011100010000000
10000101001000000110
01100110001001001111
11111011110111100011
11010011110000100010
01100101110101100010
10100000101000011010
11101001111111010011
10010000101110000110
11010011101011101000
11010101111110001110
01000001101111101110
00111111011110101110
11110001000111110111
01000101000000101000
11010000010010110111
11000100011111101110
00101001011111111101
10011011111111001110
10100001011011110111
10011100111010100010
11110000101011101101
00000101111000000110
11100111111111000110
10010101001111101111
01010001011101111111
10001100011111111111
10000101010010100010
00100001111010111001
10011000111010011110
10110001001010100111
01001100101000100010
11111101111000111111
00110000110001000011
10000000000000000010
10111011111000101011
10110100011000101010
10000101101111000101
00000000001111010110
01100101011111111111
10111111011111111111
01111001111110101011
00000001000001100011
01010101111010010011
01101111111011100010
00000101000010000010
10000101011111111110
10001100111000100010
11000000001101110111
11001100000001001101
10110111100101100111
01100101001101101111
10100000101100000010
00100011110100111111
10111111110011000011
10101101011010010000
00000010001000100000
00100100101101101101
01000001011110111110
