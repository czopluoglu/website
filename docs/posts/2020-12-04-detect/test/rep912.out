-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep912.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.33
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  3
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8905
                   IDN index value:   0.7526
                           Ratio r:   0.6698

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    1    1    1    1    2    1    1
    3    3    3    3    2    3    3    3    1    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    4    5    6    7    9   10   19

   -----------CLUSTER  2-------------

    2    3    8   15

   -----------CLUSTER  3-------------

   11   12   13   14   16   17   18   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+-+++--++--+-------
   +d+-++++-+----------
   -+d++++----++-+-+---
   +-+d+-+++-++--+-----
   ++++d++-+++----+----
   +++-+d+-----+--+----
   -+++++d+----+-+-+---
   -+-+--+d-----++-++--
   +--++---d+++-+-----+
   ++--+---+d++----+--+
   ---++---++d+-------+
   --++----+++d+++--++-
   +-+--++----+d++-+++-
   -------++--++d++++++
   --++--++---+++d++++-
   ----++-------++d+-++
   --+---++-+--++++d+++
   -------+---++++-+d++
   -----------+++++++d+
   --------+++--+-++++d
 
 No cross validation for this DETECT run
